### PR TITLE
RIA-5448-ai-access-and-ho

### PR DESCRIPTION
**Pull request by rajeshthuraiyur**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RIA-5448

### Change description ###
Non-production kustomize changes for the below ia-components
ia-case-access-api
ia-timed-event-service

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: k8s/demo/common-overlay/ia/kustomization.yaml [hunk 1]
```diff
@@ -11,7 +11,8 @@ patchesStrategicMerge:
 - ../../../namespaces/ia/ia-case-api/demo.yaml
 - ../../../namespaces/ia/ia-bail-case-api/demo.yaml
 - ../../../namespaces/ia/ia-case-notifications-api/demo.yaml
-- ../../../namespaces/ia/ia-case-documents-api/demo.yaml
```
Comment (thomast1906) [inline k8s/demo/common-overlay/ia/kustomization.yaml:14]: Why removal of` ia-case-documents-api` ?

File: k8s/ithc/common-overlay/ia/kustomization.yaml [hunk 2]
```diff
@@ -12,7 +12,8 @@ patchesStrategicMerge:
 - ../../../namespaces/ia/ia-case-payments-api/ithc.yaml
 - ../../../namespaces/ia/ia-bail-case-api/ithc.yaml
 - ../../../namespaces/ia/ia-case-notifications-api/ithc.yaml
-- ../../../namespaces/ia/ia-case-documents-api/ithc.yaml
```
Comment (thomast1906) [inline k8s/ithc/common-overlay/ia/kustomization.yaml:15]: Why removal of` ia-case-documents-api` ?

File: k8s/demo/common-overlay/ia/kustomization.yaml (hunk 1 above)
Comment (thomast1906) [reply k8s/demo/common-overlay/ia/kustomization.yaml:14]: discussed with Rajesh, duplicate entry

File: k8s/ithc/common-overlay/ia/kustomization.yaml (hunk 2 above)
Comment (thomast1906) [reply k8s/ithc/common-overlay/ia/kustomization.yaml:15]: discussed with Rajesh, duplicate entry

**Review by thomast1906 — APPROVED** [review]

File: k8s/ithc/common-overlay/ia/kustomization.yaml (hunk 2 above)
Comment (rajeshthuraiyur) [reply k8s/ithc/common-overlay/ia/kustomization.yaml:15]: its a duplicate entry

File: k8s/demo/common-overlay/ia/kustomization.yaml (hunk 1 above)
Comment (rajeshthuraiyur) [reply k8s/demo/common-overlay/ia/kustomization.yaml:14]: its a duplicate entry

**Review by augustio — APPROVED** [review]

**Pull request merged**